internal/config/config.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import (
+	"github.com/spf13/viper"
+)
+
+type DbConfig struct {
+	Host     string `mapstructure:"DB_HOST"`
+	Port     string `mapstructure:"DB_PORT"`
+	Username string `mapstructure:"DB_USERNAME"`
+	Password string `mapstructure:"DB_PASSWORD"`
+	Database string `mapstructure:"DB_DATABASE"`
+}
+
+var dbConfig DbConfig
```

Comment:
This probably won't be only dbconfig in the future? Since we are loading all env vars into this. Maybe change to Config.DBConfig or something like that.